data.c

```diff
@@ -525,11 +515,59 @@ backup_data_file(const char *from_root,
 
 		file->read_size += read_len;
 	}
+	/*
```

Comment:
I want to known when an 'odd' page appear.
This question is not about current diff.

Reply:
I think it's for a case when PostgreSQL is changing the table size and the process has not completed.

Reply:
Pg_rman read file on disk and pg flush data with size BLCKSZ.
How it will appear.
It is related to when system buff flush？

Reply:
I think pg_rman can read a page during PostgreSQL writing the page until BLCKSZ.

Reply:
OK~~